src/pages/v2/p/[projectId]/components/V2V3Dashboard.tsx

```diff
@@ -19,20 +18,15 @@ export function V2V3Dashboard({
   return (
     <div style={layouts.maxWidth}>
       <TransactionProvider>
-        <V2V3ContractsProvider>
-          <V2V3ProjectMetadataProvider
-            projectId={projectId}
-            metadata={metadata}
-          >
-            <V2V3ProjectProvider projectId={projectId}>
-              <NftRewardsProvider>
-                <VeNftProvider projectId={projectId}>
-                  <V2V3Project />
-                </VeNftProvider>
-              </NftRewardsProvider>
-            </V2V3ProjectProvider>
-          </V2V3ProjectMetadataProvider>
-        </V2V3ContractsProvider>
+        <V2V3ProjectMetadataProvider projectId={projectId} metadata={metadata}>
```

Comment:
This diff is moving the `V2V3ProjectMetadataProvider` to the parent component (which is the index.page.js).